pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -64,10 +62,6 @@ const (
 	nsObjectName       = "namespaces.json"
 	metadataObjectName = "metadata.json"
 
-	backupCancelBackoffInitialDelay = 5 * time.Second
-	backupCancelBackoffFactor       = 1
```

Comment:
Are these removed coz of golan update?

Reply:
Yes, StaticCheck was giving Unused variable errors.